internal/checker/terraform/rules/gcp/google_dataflow_job_private_job.rego

```diff
@@ -0,0 +1,54 @@
+# METADATA
+# title: "Verify Dataflow Jobs Privacy"
+# description: "This check ensures that Google Cloud Dataflow jobs are configured to be private. Private Dataflow jobs restrict access to authorized users, minimizing the risk of unauthorized access to sensitive data and job configurations."
+# related_resources:
+# - https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/spanner_database
+# custom:
+#   id: CB_TFGCP_098
+#   severity: MEDIUM
```

Comment:
update severity to high?

Reply:
Noted on this one, I will update it to high